src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -113,8 +70,6 @@ const LandingPage = ({
 
 LandingPage.propTypes = {
   data: PropTypes.object,
-  recordEvent: PropTypes.func,
-  showVerifyAndRegisterAlert: PropTypes.func,
 };
 
 export default LandingPage;
```

Comment:
Removes the concerns of alerts from `<LandingPage />`, which are now encapsulated by `<Alerts />`